database/models.py

```diff
@@ -92,6 +92,27 @@
 )
 
 
+legal_effects_association = Table(
+    "legal_effects_association",
+    Base.metadata,
+    Column(
```

Comment:
Both fields should be marked as not nullable. When not using typed declared tables this must be specified manually.